doc/design.md

```diff
@@ -152,22 +152,24 @@ Creates a user
 - format: `JSON`
 - fields:
     * email
-    * encrypted password
-    * admin permissions
+    * password
+    * admin
 
 Returns: New user
 
 - format: `JSON`
 - fields:
     * email
-    * admin permissions
+    * admin
     * timestamp
```

Comment:
user_id I should be in the returned fields, I think. (Sorry, should have thought of this during the first review.)

Reply:
Sure I can add this. I think earlier we (someone?) had decided the frontend doesn't need to see the user_id. I'm curious what any uses would be, but I can add this because doing so doesn't seem to be a problem.

Reply:
Oh yeah, if the frontend doesn't need to see it, then it's probably best to leave it out. But there should be a comment saying that this is an intentional decision due to the design of the app: no user needs to see the ID.

The main usecase I can think of is for admin users: if they see a list or table of other users, it would be correct to use the user_id as a primary key on the frontend as well. And if admin users can do update or delete operations on other users then they'll need the user_id for those.